go/pkg/pwapi/api_team-create.go

```diff
@@ -93,6 +96,20 @@ func (svc *service) TeamCreate(ctx context.Context, in *TeamCreate_Input) (*Team
 		organizationID = organization.ID
 	}
 
+	// check that user is member of the organization
```

Comment:
that's only if an organization with the same name already existed before

in that case, yes you need to belong to the organization

in the other case, creating the team will also create a new organization in a lazy way

Reply:
If you check a few lines upper, the organization is automatically created if not existing, however if one organization with this slug/id exists then user needs to be a member, isn't it fine ?

Reply:
ah yes cool, so:

1. put this in a transaction
2. add a `else` ot only check if the user is member of the organization if we didn't created a new org
3. yep, looks good for the rest, only check if user in the org with same slug :+1: